### PR TITLE
[wip] Dplus support content update

### PR DESCRIPTION
**Pull request by rtgoodwin**
(Sorry this wasn’t branched!)

### CODE REVIEW

**Comment by ktbartholomew** [comment]
@rtgoodwin Please move `support/faq.html` to `support/faq/index.html` and we'll be good to go.

**Comment by rtgoodwin** [comment]
Moved!

File: support/index.html [hunk 1]
```diff
@@ -2,56 +2,23 @@
...
+</p>
+<p>
+  If you need help building or managing your services with our tools and APIs, you can always reach out to us via:
+</p
 
-      <p class="lead">
-        Customers on the developer+&trade; tier can contact support directly twice per month by requesting a call from a Racker. <br />
-        In the meantime, you can always:<br />
-      </p>
+<!-- META-KEITH-ACTIVATE: Just 3 stacked rows: icon - service - detail/link -->
```
Comment (rgbkrk) [inline support/index.html]: Ha

Comment (rtgoodwin) [inline support/index.html]: :) He said to use descriptive tags! We are meeting on this topic this morning so hold off on merge for now.
Richard Goodwin
Product Manager: Mobile Apps & Experience, Developer Experience
[http://600a2794aa4ab5bae6bd-8d3014ab8e4d12d3346853d589a26319.r53.cf1.rackcdn.com/signatures/images/rackspace_logo.png]

richard.goodwin@rackspace.com
phone: 512-788-5403
mobile: 512-736-7897
Skype: rtgoodwin<file:///callto/::rtgoodwin> - Yahoo: richardtgoodwin<file:///ymsgr/sendim%3Frichardtgoodwin>
AIM: dellovision - IRC: goody / rgoodwin
Ideation | Intellection | Activator | Relator | Responsibility

Comment (smashwilson) [inline support/index.html]: > hold off on merge for now.

:+1:

I've edited the pull request title to indicate that it's work-in-progress. Just take off the `[wip]` once it's ready to go and we'll :shipit:

**Comment by kimtryce** [comment]
I'm thinking about killing this as we killed the D+ program.  @everett-toews @smashwilson thoughts?

**Comment by everett-toews** [comment]
👍

**Pull request closed without merging**